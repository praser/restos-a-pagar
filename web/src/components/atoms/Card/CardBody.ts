import styled from 'styled-components';

const CardBody = styled.div`
  flex: 1 1 auto;
  padding: 1.25rem;
`;

export default CardBody;
