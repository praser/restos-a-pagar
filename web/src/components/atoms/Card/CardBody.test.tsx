import React from 'react';

import renderer from 'react-test-renderer';

import CardBody from './CardBody';

describe('CardBody atom component', () => {
  it('is expected to render the component correctly', () => {
    const cardBody = renderer.create(<CardBody />).toJSON();
    expect(cardBody).toMatchSnapshot();
  });
});
