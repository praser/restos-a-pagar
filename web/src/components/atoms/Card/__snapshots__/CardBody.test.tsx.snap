// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardBody atom component is expected to render the component correctly 1`] = `
<div
  className="sc-bdfBwQ ecyRgt"
/>
`;
